openslides_backend/presenter/get_users.py

```diff
@@ -0,0 +1,133 @@
+from operator import itemgetter
+from typing import Any, Dict, List
+
+import fastjsonschema
+
+from ..services.datastore.deleted_models_behaviour import DeletedModelsBehaviour
+from ..shared.exceptions import PresenterException
+from ..shared.patterns import Collection
+from ..shared.schema import schema_version
+from .base import BasePresenter
+from .presenter import register_presenter
+
+get_users_schema = fastjsonschema.compile(
+    {
+        "$schema": schema_version,
+        "type": "object",
+        "title": "get_users",
+        "description": "get user ids data",
+        "properties": {
+            "start_index": {"type": "integer"},
+            "entries": {"type": "integer"},
+            "sort_criteria": {
+                "type": "array",
+                "item": {
+                    "type": "string",
+                    "enum": ["username", "first_name", "last_name"],
+                },
+            },
+            "reverse": {"type": "boolean"},
+            "include_temporary": {"type": "boolean"},
+            "filter": {"type": ["string", "null"]},
+        },
+        "required": [],
+        "additionalProperties": False,
+    }
+)
+
+ALLOWED = {
+    "first_name": "",
+    "last_name": "",
+    "username": "",
+    "title": "",
+    "is_active": False,
+    "is_physical_person": False,
+    "email": "",
+    "last_email_send": "",
```

Comment:
I think that this is a number - use 0 as a default. The client often wants to know about not-sended emails, so sorting them first sounds good.

Reply:
No, in the models.py stands: it is a CharField.

Reply:
Ok, this is a mistake. I fixed it here: https://github.com/OpenSlides/OpenSlides/pull/5912

Reply:
Updated this.